### DISCUSSION TITLE
Changes/Updates

### DISCUSSION THREAD

**Discussion by ghost** [The-Forum-Participators/the-forum-participators.github.io#6] [Ideas]
If you want to suggest a change or update, do it here!

**Comment by Jbear-Scratch** [comment]
- [x] On the website, could you add a topic under contact us, under a section called questions?Then put : please ask questions in this forum post (https://scratch.mit.edu/discuss/topic/585879/?page=1#post-6097241) or on our group comments (https://scratch.mit.edu/studios/31153550/comments)

**Comment by ghost** [comment]
@Jbear-Scratch Done
